docs/basics/server.md

```diff
@@ -105,8 +102,18 @@ gremlin> graph.addVertex("name", user)
 No such property: user for class: Script21
 Type ':help' or ':h' for help.
 Display stack trace? [yN]
+```
+
+Alternately, you can establish a consolewith a session by specifying
+[session](https://tinkerpop.apache.org/docs/current/reference/#sessions)
+when creating the connection. A [console
+session](https://tinkerpop.apache.org/docs/current/reference/#console-sessions)
```

Comment:
I know we had this link here before already, but it's best to use `{{ tinkerpop_version }}` instead of `current` in links to TinkerPop docs to make them always point to the correct version of the TinkerPop docs.

Reply:
Yep, looks like a good opportunity now to replace the link